### ISSUE TITLE
windows release x64 make

### ISSUE THREAD

**Issue by MichaelWallace30** [attie/libxbee3#14]
I followed your website instructions for compiling as x64 in windows 7. It compiles fine, yet x64 DLL projects fail to link with x64 libxbee3.dll The compiled x64 libxbee3.dll links fine with x86 DLL projects. I assume the make file is the issue. I tried multiple times. The issue was solved by compiling libxbee3 in visual studio.

**Comment by attie** [comment]
Hi Michael,

Unfortunately I don't often find myself with a functioning Windows dev
environment, but I'll look into this when I find a moment.
I'm glad you managed to find a work around for now.

Attie

**Comment by MichaelWallace30** [comment]
Yeah I'm in no rush and by you code comments I know you love Windows.
